### PR TITLE
Add SUSE Linux Enterprise Micro in the external reporting

### PR DESCRIPTION
**Pull request by jlausuch**
We have SLE Micro product which is different than OpenSUSE MicroOS

### CODE REVIEW

**Comment by jlausuch** [comment]
@okurz

**Review by okurz — APPROVED** [review]

**Comment by jlausuch** [comment]
@okurz I don't have rights to merge, I see some CI failures, but I guess recurrent and not related to this change. Could you please merge if appropriate?

File: templates/webapi/branding/openSUSE/external_reporting.html.ep [hunk 1]
```diff
@@ -15,13 +15,15 @@
     openqa => 'https://progress.opensuse.org/projects/openqav3/issues/new',
     kubic => 'https://bugzilla.opensuse.org/enter_bug.cgi',
     microos => 'https://bugzilla.opensuse.org/enter_bug.cgi',
+    sle-micro => 'https://bugzilla.suse.com/enter_bug.cgi',
```
Comment (Martchus) [inline templates/webapi/branding/openSUSE/external_reporting.html.ep:18]:
```suggestion
    'sle-micro' => 'https://bugzilla.suse.com/enter_bug.cgi',
```

File: templates/webapi/branding/openSUSE/external_reporting.html.ep [hunk 2]
```diff
@@ -15,13 +15,15 @@
...
 );%>
 <% my %distri_to_prod = (
     sle => 'SUSE Linux Enterprise',
     opensuse => 'openSUSE',
     caasp => 'SUSE CaaS Platform',
     kubic => 'openSUSE',
     microos => 'openSUSE',
+    sle-micro => 'SUSE Linux Enterprise Micro',
```
Comment (Martchus) [inline templates/webapi/branding/openSUSE/external_reporting.html.ep:26]:
```suggestion
    'sle-micro' => 'SUSE Linux Enterprise Micro',
```

**Review by Martchus — CHANGES_REQUESTED** [review]
The CI failures are related:

```
[error] [JTtZJqbD] Bareword "sle" not allowed while "strict subs" in use at template branding/openSUSE/external_reporting.html.ep line 18.
Context:
  13:     opensuse => 'https://bugzilla.opensuse.org/enter_bug.cgi',
  14:     caasp => 'https://bugzilla.suse.com/enter_bug.cgi',
  15:     openqa => 'https://progress.opensuse.org/projects/openqav3/issues/new',
  16:     kubic => 'https://bugzilla.opensuse.org/enter_bug.cgi',
  17:     microos => 'https://bugzilla.opensuse.org/enter_bug.cgi',
  18:     sle-micro => 'https://bugzilla.suse.com/enter_bug.cgi',
  19: );%>
```

File: templates/webapi/branding/openSUSE/external_reporting.html.ep (hunk 1 above)
Comment (jlausuch) [reply templates/webapi/branding/openSUSE/external_reporting.html.ep:18]: and what about all other entries?

**Review by Martchus — APPROVED** [review]

**Review by kalikiana — APPROVED** [review]

**Comment by okurz** [comment]
@jlausuch commonly we don't force merge if we just *think* a CI error might be unrelated :) I am convinced that this is a recipe for failure and rather bad habit. Take every CI error serious. Sure, there can be unrelated problems but I suggest to always handle CI problems before accepting them as "unrelated". Of course, that is certainly the responsibility of the main contributors which can always be asked for help.

@Martchus @kalikiana are the coverage changes in t/lib/OpenQA/SeleniumTest.pm and t/lib/OpenQA/Test/Utils.pm known and planned accordingly in tickets?

**Comment by kalikiana** [comment]
> @Martchus @kalikiana are the coverage changes in t/lib/OpenQA/SeleniumTest.pm and t/lib/OpenQA/Test/Utils.pm known and planned accordingly in tickets?

Covered by [poo#80800](https://progress.opensuse.org/issues/80274) me thinks. I'll propose something

**Pull request merged**